### DISCUSSION TITLE
HW3 Part 2

### DISCUSSION THREAD

**Discussion by JeffHann** [cooperrc/computational-mechanics#113] [Q&A]
After we plot Height vs. Time for the lacrosse and tennis balls with the fall drag functions, are we supposed to plot the balls' positions using x and y? The assignment shows the 2D FBD for a projectile and says to "Consider the 2D motion of the lacrosse ball" -> is that asking us to plot the xy positions and/or velocities for the lacrosse ball from the previous part?
I don't see an explicit question or exercise statement, so I just want to clarify if there is something we need to to do here or if this section is just for information.
<img width="497" alt="image" src="https://user-images.githubusercontent.com/112503497/195204169-7a538c6d-a83d-4e0e-8499-bd5ed166b671.png">

**Comment by saewon1** [comment]
I assumed it was just for discussion's sake but i could be wrong
